test/es-module/test-esm-loader-modulemap.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+// Flags: --expose-internals
+
+// This test ensures that the type checking of ModuleMap throws errors appropriately
+
+const { URL } = require('url');
+
+const common = require('../common');
```

Comment:
Please make this the first `require()` statement (just move it before `require('url')`).